commuting/src/commuterCategorize.r

```diff
@@ -122,86 +171,217 @@ drivingDistance <- function(origin,destination){
 }
 
 
-# new distance function:
-getDistance <- function(fcdr, towers) {
-  dist <- group_by(fcdr, ANUMBER) %>% summarise(distCommute = drivingDistance(getCoords(getHomeID(fcdr, ANUMBER), towers), getCoords(getWorkID(fcdr, ANUMBER), towers)))
+get_distance <- function(fcdr, towers) {
+    # Creates a data frame containing the distances between home and work
+    # towers.
+    #
+    # Args:
+    #   fcdr: The filtered cdr.
+    #   towers: The data frame consisting of all tower data. 
+    #
+    # Returns:
+    #   The data frame of fcdr with an added column of driving distances
+    #   between home and work.
+  
+    dist <- group_by(fcdr, ANUMBER) %>% 
+	summarise(distCommute = driving_distance(get_coords(get_home_id(k_options$HOME_TYPE, fcdr, ANUMBER), towers), get_coords(get_work_id(fcdr, ANUMBER), towers)))
   return(dist)
 }
 
 
+show_home_and_work_towers <- function(data, towers, k_options) {
+    # Creates a data frame which shows the home/work towers for each
+    # ANUMBER.
+    #
+    # Args:
+    #   data: The cdr.
+    #   towers: The tower data.
+    #   k_options: The data frame containing a set of user-defined
+    #              constants.
+    #
+    # Returns:
+    #   The data frame with all home and work towers for each ANUMBER.
+  
+    probable_place <- group_by(data, ANUMBER, START_DATE_TIME, CELL_ID) %>% 
+    
+	summarise(PLACE = home_or_work(START_DATE_TIME, k_options)) %>%
+	group_by(ANUMBER) %>%
+	summarise(HOME_ID = get_mode_by_label(CELL_ID, PLACE, "Home"),
+	      WORK_ID = get_mode_by_label(CELL_ID, PLACE, "Work")) %>%
+	return()
+}
+
+
+show_home_by_label <- function(data, towers, k_options) {
+    # Shows the type of home_id based on user-defined constant.
+    #
+    # Args:
+    #   data: The cdr.
+    #   towers: The tower data.
+    #   k_options: The data frame containing a set of user-defined
+    #              constants.
+    #
+    # Returns:
+    #   The filtered cdr with the homes and works shown for each ANUMBER. 
```

Comment:
It currently only shows home locations. I fixed the comment on line 225.
As a TODO, this function could be modified and used to show either homes and works or only homes, rather than have two separate functions.